go/utils/helper.go

```diff
@@ -0,0 +1,45 @@
+package utils
```

Comment:
Another way is to create a utils folder, but put different utils into packages under the folder: https://github.com/coreos/etcd/tree/master/pkg . Otherwise everything will be all under package `utils`, we don't get the benefit of encapsulation between packages.